app/styles/_utils.less

```diff
@@ -16,3 +16,8 @@
 .hide-if-empty:empty {
   display: none !important;
 }
+
+// generic classes not included with twbs
+.inline-block {
+  display:inline-block;
+}
```

Comment:
Not sure this is worthwhile. How is it better than `style="display: inline-block;"`? It's basically an inline style that we've given a class name.

@rhamilto Thoughts?